src/hotspot/os/posix/os_posix.cpp

```diff
@@ -891,6 +891,10 @@ void os::funlockfile(FILE* fp) {
   ::funlockfile(fp);
 }
 
+char* os::realpath(const char* filename, char* outbuf, size_t outbuflen) {
+  return os::Posix::realpath(filename, outbuf, outbuflen);
+}
```

Comment:
We don't need `os::Posix::realpath` any more - just rename it to `os::realpath`.